code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -387,13 +387,16 @@
 	to_chat(user, span_notice("Controls are now [bot_cover_flags & BOT_COVER_LOCKED ? "locked" : "unlocked"]."))
 	return TRUE
 
+/mob/living/simple_animal/bot/screwdriver_act(mob/living/user, obj/item/tool)
+	if(!(bot_cover_flags & BOT_COVER_LOCKED))
+		bot_cover_flags ^= BOT_COVER_OPEN
+		to_chat(user, span_notice("The maintenance panel is now [bot_cover_flags & BOT_COVER_OPEN ? "opened" : "closed"]."))
+	else
+		to_chat(user, span_warning("The maintenance panel is locked!"))
+
 /mob/living/simple_animal/bot/attackby(obj/item/attacking_item, mob/living/user, params)
 	if(attacking_item.tool_behaviour == TOOL_SCREWDRIVER)
```

Comment:
just delete these two lines instead of having it return

Reply:
It seems necessary; without it, you still smack the mulebot with the screwdriver since it falls through to the parent call

Reply:
have the screwdriver_act return TRUE

Reply:
Should be fixed now, wasn't aware of how the chain worked. Pretty neat.